TestApp/TestApp/AppDelegate.swift

```diff
@@ -17,7 +17,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-        BrandingManager.set(brand: FractalBrand())
+        BrandingManager.set(brand: LearningBrand())
```

Comment:
i'm guessing i'm not supposed to merge this at this point but obviously LearningBrand isn't a good name and shouldn't be the default for my master branch.